### PR TITLE
Enable and deflake oom test

### PR DESCRIPTION
**Pull request by jbartosik**
Includes #3017 and one a commit increasing timeout for the OOM test

### CODE REVIEW

File: vertical-pod-autoscaler/e2e/v1/full_vpa.go [hunk 1]
```diff
@@ -91,37 +92,80 @@ var _ = FullVpaE2eDescribe("Pods under VPA", func() {
...
+		vpaClientSet = vpa_clientset.NewForConfigOrDie(config)
+		vpaClient := vpaClientSet.AutoscalingV1()
+		_, err = vpaClient.VerticalPodAutoscalers(ns).Create(vpaCRD)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	})
+
+	ginkgo.It("have memory requests growing with OOMs", func() {
+		listOptions := metav1.ListOptions{LabelSelector: "name=hamster", FieldSelector: getPodSelectorExcludingDonePodsOrDie()}
+		err := waitForResourceRequestInRangeInPods(
+			f, 8*time.Minute, listOptions, apiv1.ResourceMemory,
```
Comment (krzysied) [inline vertical-pod-autoscaler/e2e/v1/full_vpa.go:161]: Now that I think about it, I would like to see a const for the timeout.

File: vertical-pod-autoscaler/e2e/v1/full_vpa.go [hunk 2]
```diff
@@ -91,37 +92,80 @@ var _ = FullVpaE2eDescribe("Pods under VPA", func() {
...
+		})
+
+		vpaClientSet = vpa_clientset.NewForConfigOrDie(config)
+		vpaClient := vpaClientSet.AutoscalingV1()
+		_, err = vpaClient.VerticalPodAutoscalers(ns).Create(vpaCRD)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	})
+
+	ginkgo.It("have memory requests growing with OOMs", func() {
+		listOptions := metav1.ListOptions{LabelSelector: "name=hamster", FieldSelector: getPodSelectorExcludingDonePodsOrDie()}
```
Comment (krzysied) [inline vertical-pod-autoscaler/e2e/v1/full_vpa.go:159]: nit. Long line, please break it. Sorry I didn't catch it before...

**Comment by krzysied** [comment]
Btw, I'm ok with having this one PR (with different title) instead of two separate - #3017 can be closed.

File: vertical-pod-autoscaler/e2e/v1/full_vpa.go (hunk 1 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/e2e/v1/full_vpa.go]: Done

File: vertical-pod-autoscaler/e2e/v1/full_vpa.go (hunk 2 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/e2e/v1/full_vpa.go]: Done, PTAL

**Pull request merged**